minari/dataset/minari_storage.py

```diff
@@ -247,10 +247,11 @@ def update_from_buffer(self, buffer: List[dict], data_path: str):
 
                 # TODO: save EpisodeMetadataCallback callback in MinariDataset and update new episode group metadata
 
-            file.attrs.modify("total_episodes", last_episode_id + len(buffer))
-            file.attrs.modify(
-                "total_steps", file.attrs["total_steps"] + additional_steps
-            )
+            self._total_steps = file.attrs["total_steps"] + additional_steps
+            self._total_episodes = last_episode_id + len(buffer)
+
+            file.attrs.modify("total_episodes", self._total_episodes)
+            file.attrs.modify("total_steps", self._total_steps)
```

Comment:
Thanks for these fixes also. I am wondering if we need `self._total_steps` or if we can just make the `@property` total_steps reading from the file (and same for total_episodes)

Reply:
I think it might marginally save on file reads if we keep it the way it is for some use-cases, but it doesn't seem like it would make a super big difference either way.